data-updates.lua

```diff
@@ -9,6 +9,10 @@ elseif settings.startup["electric-locomotive-speed-setting"].value == "378 km/h"
   data.raw["locomotive"]["electric-locomotive"].max_speed = 1.75
   data.raw["cargo-wagon"]["electric-cargo-wagon"].max_speed = 1.75
   data.raw["fluid-wagon"]["electric-fluid-wagon"].max_speed = 1.75
+elseif settings.startup["electric-locomotive-speed-setting"].value == "714 km/h (Extended)" then
+  data.raw["locomotive"]["electric-locomotive"].max_speed = 3.0
+  data.raw["cargo-wagon"]["electric-cargo-wagon"].max_speed = 3.0
+  data.raw["fluid-wagon"]["electric-fluid-wagon"].max_speed = 3.0
```

Comment:
Forgive me if my maths is wrong - the default speed has a 1.1 multiplier, and 518 is 2.4 (See `building-and-stock.lua` line 19)

518/2.4 * 3 is roughly 647, not 714. This probably needs to be 3.3? What does it say in game over the entity when you test it?

Reply:
Well, if `238 = 1.1` is vanilla and I want to add a 3x multiplier to that, that certainly has to be `3.3` :D

I will fix that!